CMakeLists.txt

```diff
@@ -156,12 +156,38 @@ else()
     message(FATAL_ERROR "Could not find msgpack-cxx")
 endif()
 
-# Add abseil-cpp
+# Include dependency settings.
+include("build/deps/cmake-settings/all.cmake")
```

Comment:
Can we:
1. move this as close to the top as possible (so it is unlikely we try to use something from the include before it)
2. copy the block in https://github.com/y-scope/ystdlib-cpp/blob/main/CMakeLists.txt#L42